components/02-molecules/alert/_yds-alert.scss

```diff
@@ -24,7 +31,9 @@ $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
   }
 
   &[data-alert-state='dismissed'] {
-    display: none;
+    opacity: 0;
+    max-height: 0;
+    padding: 0;
```

Comment:
Dang! I forgot one last thing. We're visually hiding this, but one other thing `display: none` was doing was "hiding" it from accessible tech, like screen readers. Right now, the alert would still be read to the user, and stuff. So we need to add the `animate-hidden` mixin to remove it from assistive tech when it is visually hidden.

Adding `@include tokens.animate($alert-fade-speed);` inside the "dismissed" state should do it.

Reply:
Also, I don't think we need this `padding: 0` ... at least, I couldn't figure out what it was doing.

Reply:
Oh! I bet that was a carry-over from when we had these styles on the `inner` element. I think it can be safely removed now.

Reply:
@ModulesUnraveled I pushed up a couple commits addressing these. :crossed_fingers: Should be good once it's done running tests.